{
    "comments": [
        {
            "author": null,
            "body": "Well, that's a known issue. None of the chars which aren't allowed in a filename\nare filtered out at the moment, so ...\nI think this is the same behavior as with TC 4.1.x.",
            "date": "20030623T13:33:02",
            "id": 0
        },
        {
            "author": null,
            "body": "Actually, small correction: this doesn't happen in TC 4.1.x, as it doesn't try\nto save the context stuff as separate files.\nI'll slightly refactor things and expose a config file base field at the host\nlevel, and add escaping code in StandardContext. This should fix the issue, as\nwell as bug 19607.",
            "date": "20030623T14:05:30",
            "id": 1
        },
        {
            "author": null,
            "body": "*** Bug 20217 has been marked as a duplicate of this bug. ***",
            "date": "20030625T08:26:34",
            "id": 2
        },
        {
            "author": null,
            "body": "This should have been fixed as part of the deployer refactorings.\nThe context file name should now be decoupled from the path name. In case a\ncontext file doesn't exist (so it will be created automatically when saving\nchanges in the admin webapp), '/' will be escaped.",
            "date": "20030625T08:29:23",
            "id": 3
        }
    ],
    "component": "Catalina",
    "description": "Error saving (\"Commit changes\") new context with '/' char\ninto his name, for example i need to create \"/app/bpb\" context.\n\nWorkaround:\nI create \"/appbpb\" context and then I modify\nthe path attribute from \"/appbpb\" to \"/app/bpb\" \ninto the \"<HOME>/conf/Catalina/localhost/appbpb.xml\"",
    "hasPatch": false,
    "hasScreenshot": false,
    "id": "21010",
    "issuetypeClassified": "BUG",
    "issuetypeTracker": "BUG",
    "priority": "P3 normal",
    "product": "Tomcat 5",
    "project": "TOMCAT",
    "summary": "Error creating new context",
    "systemSpecification": true,
    "version": "5.0.3"
}